internal/cli/clients/client.go

```diff
@@ -1464,21 +1467,14 @@ func uniqueStrings(stringSlice []string) []string {
 	return list
 }
 
-func getCredentials(ctx context.Context) (string, string, error) {
-	mainDomain, err := domain.MainDomain(ctx)
-	if err != nil {
-		return "", "", err
-	}
-
-	if !strings.Contains(mainDomain, "omg.howdoi.website") {
```

Comment:
I wonder what the reason for this check was. Anyway, this PR looks ok, unless I'm missing something.

Reply:
Best guess based on who I remember writing the #501, a copy pasta error.